apps/docs-app/app/styles/marketing-theme.css

```diff
@@ -778,7 +778,7 @@
   }
 
   .btn {
-    --bs-btn-border-radius: 999rem; /* Custom after generation */
+    /* --bs-btn-border-radius: 999rem; Custom after generation */
```

Comment:
?

Reply:
I was doing this in testing so I can remove the comment. Regardless though this border radius of 999rem is no longer used within the marketing theme